src/Raven.Client/Documents/Session/IAsyncDocumentQuery.cs

```diff
@@ -67,6 +67,13 @@ public interface IAsyncRawDocumentQuery<T> :
     {
     }
 
+    public interface IAsyncGraphQuery<T> :
+        IQueryBase<T, IAsyncGraphQuery<T>>,
+        IAsyncDocumentQueryBase<T>
+    {
+        IAsyncGraphQuery<T> With<TOther>(string alias, IRavenQueryable<TOther> query, CancellationToken token = default);
```

Comment:
I think we do not need cancellation token here

Reply:
No, we don't, that isn't executing